data/strings/sound.txt

```diff
@@ -1209,6 +1428,46 @@
     zh-Hans = 然后
     zh-Hant = 然後
 
+  [dist_direction_onto_street]
+    comment = %1$s is the distance like in_500_meters, %2$s is the direction like make_a_right_turn_street or make_a_right_turn if unavailable, %3$s is "onto" unless the turn is an exit number, %4$s is the name/number of the street/exit, and %5$s is an optional turn verb particle
+    en = %1$s %2$s %3$s %4$s
```

Comment:
Does it make sense to copypaste everything? Shouldn't the en translation work by default for all languages?

Reply:
I don't know, but if en fallback "just works" then it's a big problem for other strings where I do logic like `if (string myTrans = getTranslation("dist_direction_onto_street", locale)){ outputDistDirOntoStr(myTrans, myString); }` -- automatic fallback would say "oh sure, we have your string. You wanted English right?" when I'm explicitly checking for the string in *my* locale. And if en fallback doesn't "just work" then the answer to your question is no. I've programmed and tested this as written, I'd have to go back and check assumptions to see what happens if/when translation strings are removed or when a string for `en` is defined but not the user's language. (It just so happens that many of our edge cases are in unrelated languages like `nl` and `ja` and not "the default" `en` so the case hasn't been tested.)

Reply:
It's better not to guess, but check/test, and make it explicit in comments/documentation. Where the fallback is ok, copy-paste is not needed.

Reply:
@biodranik I've confirmed that fallback is active when `en` is specified. For this string I've deduplicated and specified that all languages inherit from `en` by default, and for `[onto]` I've specified that an empty string is required for some languages in order to prevent nonsense. (generate_localizations would create strings like "一キロ先 Broadway Avenue onto 右折です" otherwise, where Japanese requires no "onto" or equivalent in this position, especially not in English)